app/models/manageiq/providers/openstack/manager_mixin.rb

```diff
@@ -304,6 +314,8 @@ def verify_credentials(args)
     def raw_connect(password, params, service = "Compute")
       if params[:event_stream_selection] == 'amqp'
         amqp_available?(password, params)
+      elsif params[:event_stream_selection] == 'stf'
```

Comment:
does it make sense to use a case statement now that it's more than a single if/else?

```ruby
case params[:event_stream_selection]
when "amqp" then amqp_available?(password, params)
when "stf"  then stf_available?(password, params)
else             ems_connect?(password, params, service)
end
```

Reply:
The ```case``` statement makes more sense :+1:  I prefer refactor this method in follow-up PR.

Reply:
https://github.com/ManageIQ/manageiq-providers-openstack/pull/590